backend/repositories/eventRepository.js

```diff
@@ -0,0 +1,87 @@
+var connection = require('../db/dbConnect');
```

Comment:
server connects with db during its start and connec inside repository isn't required